lib/beaker/host_prebuilt_steps.rb

```diff
@@ -343,8 +343,10 @@ def enable_root_login host, opts
         if host['platform'] =~ /osx/
           host.exec(Command.new("sudo sed -i '' 's/#PermitRootLogin no/PermitRootLogin Yes/g' /etc/sshd_config"))
           host.exec(Command.new("sudo sed -i '' 's/#PermitRootLogin yes/PermitRootLogin Yes/g' /etc/sshd_config"))
-        else
+        elsif host['platform'] !~ /windows/
```

Comment:
I don't think that you want to exclude all windows here, just non-cygwin based.  So, should be an 'if host.is_cygwin?'.